subseas_flex_fcst/predictions.py

```diff
@@ -8,7 +8,7 @@
 import pandas as pd
```

Comment:
I suggest renaming predictions.py to cpt.py, or if there's some stuff that's specific to CPT and some that's not, separating the two into different files.

Reply:
started that. More might split when we extract from cpt functions what is purely date formating and such